addons/website_event_track/controllers/main.py

```diff
@@ -129,7 +131,7 @@ def event_track_proposal(self, event, **post):
         if not event.can_access_from_current_website():
             raise NotFound()
 
-        return request.render("website_event_track.event_track_proposal", {'event': event})
+        return request.render("website_event_track.event_track_proposal", {'event': event, 'main_object': event})
```

Comment:
This doesn't seem to be necessary right ?

Reply:
Without it the tab name isn't set as we want unfortunately